### PR TITLE
Fix typo in Mesh level of detail (LOD)

### PR DESCRIPTION
**Pull request by Calinou**
This fixes https://www.reddit.com/r/godot/comments/109b4ep/were_announcing_a_godot_40_documentation_sprint/j3y3fs9/.

### CODE REVIEW

**Review by skyace65 — CHANGES_REQUESTED** [review]
This part actually needs more work. Possibly confusing wording aside, the page is still innacurate with this change. There are no negative LOD values, additionally, lower values (such as .001) make the mesh use a lower level of detail for longer, and values greater than 1 make it stay more detailed longer.

**Comment by Calinou** [comment]
> This part actually needs more work. Possibly confusing wording aside, the page is still innacurate with this change. There are no negative LOD values, additionally, lower values (such as .001) make the mesh use a lower level of detail for longer, and values greater than 1 make it stay more detailed longer.

Fixed.

**Review by mhilbrunner — APPROVED** [review]

File: tutorials/3d/mesh_lod.rst [hunk 1]
```diff
@@ -133,9 +133,9 @@ camera moves away, resulting in higher performance but lower quality.
 
 If you need to perform per-object adjustments to mesh LOD, you can adjust how
 aggressive LOD transitions should be by adjusting the **LOD Bias** property on
-any node that inherits from GeometryInstance3D. Positive values will make LOD
-transitions happen sooner than usual (resulting in lower quality but higher
-performance). Negative values will make LOD transitions happen earlier than
+any node that inherits from GeometryInstance3D. Values *above* ``1.0`` will make
+LOD transitions happen sooner than usual (resulting in lower quality but higher
```
Comment (skyace65) [inline tutorials/3d/mesh_lod.rst:137]:
```suggestion
LOD transitions happen sooner than usual (resulting in higher quality but lower
```

File: tutorials/3d/mesh_lod.rst [hunk 2]
```diff
@@ -133,9 +133,9 @@ camera moves away, resulting in higher performance but lower quality.
...
+performance). Values *below* ``1.0`` will make LOD transitions happen later than
 usual (resulting in higher quality but lower performance).
```
Comment (skyace65) [inline tutorials/3d/mesh_lod.rst:139]:
```suggestion
usual (resulting in lower quality but higher performance).
```

**Review by skyace65 — CHANGES_REQUESTED** [review]

File: tutorials/3d/mesh_lod.rst (hunk 1 above)
Comment (Calinou) [reply tutorials/3d/mesh_lod.rst:137]: The currently phrasing is already correct. Lower values of LOD Bias makes meshes look *better* at a given distance, not worse.

Comment (clayjohn) [reply tutorials/3d/mesh_lod.rst:137]: The current behaviour is that values between 0 and 1 make the LOD transition happen sooner resulting in worse quality for the same distance. 

Values above one delay the LOD transition so you get higher quality at the same distance.

Comment (Calinou) [reply tutorials/3d/mesh_lod.rst:137]: > The current behaviour is that values between 0 and 1 make the LOD transition happen sooner resulting in worse quality for the same distance.
> 
> Values above one delay the LOD transition so you get higher quality at the same distance.

So the LOD Bias property works the opposite of the Mesh LOD Threshold project setting? I consider that to be pretty odd :slightly_frowning_face:

Comment (clayjohn) [reply tutorials/3d/mesh_lod.rst:137]: > So the GeometryInstance3D LOD Bias property works the opposite of the Mesh LOD Threshold project setting? I consider that to be pretty odd slightly_frowning_face
>
>In general, I expect LOD bias to work in a fashion where positive values use lower-than-needed LODs and negative values use higher-than-needed LODs (similar to mipmap LOD bias).

Yes, it is rather odd. I would expect it to work like textureLod in GLSL as well. i.e. a bias of one bumps you up one level while -1 bumps you down one level

Comment (Calinou) [reply tutorials/3d/mesh_lod.rst:137]: Indeed, I've tested this on 4.0.beta11 and what you're saying is correct.

Could we change this behavior before release? It's likely to be a pitfall people will run into.

Comment (clayjohn) [reply tutorials/3d/mesh_lod.rst:137]: I don't know. I'm not familiar enough with the LOD system to comment one why it was implemented that way

**Review by skyace65 — APPROVED** [review]

**Review by clayjohn — APPROVED** [review]

**Comment by clayjohn** [comment]
Looks great, thanks! Let's remember to discuss the behaviour of LOD with other rendering contributors sometime soon

**Pull request merged**